utils/getInvoices/tests/getInvoices.placing.test.ts

```diff
@@ -5,139 +5,892 @@ import { expect } from '@jest/globals'
+      const prevPayment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevService,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        amount: 70,
+        price: 90,
+        sum: 90,
+      })
+    })
+
+    it('should load when service = { rentPrice: 90 }, company = { inflicion: false, totalArea: 70 }, prevService = null, prevPayment = null', () => {
+      const service: Partial<IService> = {
+        rentPrice: 90,
+      }
+      const company: Partial<IRealestate> = {
+        inflicion: false,
+        totalArea: 70,
+      }
+      const prevService: Partial<IService> = null
+      const prevPayment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevService,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        amount: 70,
+        price: 90,
+        sum: 90,
+      })
+    })
 
-    const invoices = getInvoices({
-      company,
-      service,
-      payment,
-      prevPayment,
+    it('should load when service = { rentPrice: 90 }, company = { inflicion: true, totalArea: 70 }, prevService = null, prevPayment = null', () => {
+      const service: Partial<IService> = {
+        rentPrice: 90,
+      }
+      const company: Partial<IRealestate> = {
+        inflicion: true,
+        totalArea: 70,
+      }
+      const prevService: Partial<IService> = null
+      const prevPayment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevService,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+          type: ServiceType.Placing,
+          price: 6300,
+          sum: 6300,
+        })
     })
 
-    expect(invoices).not.toContainEqual(
-      expect.objectContaining({ type: ServiceType.Placing })
-    )
+    it('should load when service = { rentPrice: 90 }, company = { inflicion: true, totalArea: 70 }, prevService = { inflicionPrice: 110 } prevPayment = null', () => {
+      const service: Partial<IService> = {
+        rentPrice: 90,
+      }
+      const company: Partial<IRealestate> = {
+        inflicion: true,
+        totalArea: 70,
+      }
+      const prevService: Partial<IService> = {
+        inflicionPrice: 110,
+      }
+      const prevPayment: Partial<IPayment> = null
+
+      const invoices = getInvoices({
+        service,
+        company,
+        prevService,
+        prevPayment,
+      })
+
+      expect(invoices).toContainEqual({
+        type: ServiceType.Placing,
+        price: 6930,
+        sum: 6930,
+      })
+    })
+
+    it('should load when service = { inflicionPrice: 110, rentPrice: 90 }, company = { inflicion: true, totalArea: 70 }, prevService = { inflicionPrice: 110 } prevPayment = { invoice: [Placing] }', () => {
```

Comment:
use different `inflicionPrice` values in `service` and `prevService` to test that we properly use only `prevService` inflicion value